pkg/chunk/aws/dynamodb_table_client_test.go

```diff
@@ -20,8 +20,8 @@ import (
 const (
 	tablePrefix      = "cortex_"
 	chunkTablePrefix = "chunks_"
-	tablePeriod      = 7 * 24 * time.Hour
-	gracePeriod      = 15 * time.Minute
+	tablePeriod      = model.Duration(7 * 24 * time.Hour)
+	gracePeriod      = model.Duration(15 * time.Minute)
```

Comment:
As far as I can see this is always cast to a `time.Duration` so why not leave it as that?